### PR TITLE
JA-translation-proposal-lifecycle

### PR DESCRIPTION
**Pull request by mattletford**
Convert .docx to Markdown for JA translation of proposal lifecycle @agektmr

### CODE REVIEW

**Comment by agektmr** [comment]
@mattletford I've indicated the destination path of the file in the Google Doc file. Could you please move it to the right place?

**Comment by agektmr** [comment]
Oh you pushed the change at the same time. Never mind my previous message :)

File: site/ja/docs/privacy-sandbox/proposal-lifecycle/index.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+ディスカッションの段階は、非常に複雑になる場合があります。
+
+たとえば、[FLEDGE](https://github.com/WICG/turtledove/blob/main/FLEDGE.md)
+は、クロスサイト トラッキングを行わずにインタレスト
+ベースの広告をサポートするための提案です。FLEDGE
+は、プライバシーを強く望む人々や多くの業界関係者の意見を参考に、以前の 2
+つの提案（PIGIN と TURTLEDOVE）から進化しました。100 以上の組織が [W3C での会議](https://github.com/WICG/turtledove/issues/88)に参加して現行バージョンの改善を支援し、[オンランディスカッション スレッドも 200](https://github.com/WICG/turtledove/issues)
+を超えました。
+
+{% Img src="image/C47gYyWYVMMhDmtYSLOWazuyePF2/GUccejeZiolwdTSzwW0z.avif", alt="2019 年に PIGIN が提案され、2020 年に TURTLEDOVE、2021 年に FLEDGE が提案されました。", width="800", height="169" %}
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/proposal-lifecycle/index.md:69]:
```suggestion
{% Img src="image/VbsHyyQopiec0718rMq2kTE1hke2/Rg4lEQVzb1v6CZTyzFTE.png", alt="2019 年に PIGIN が提案され、2020 年に TURTLEDOVE、2021 年に FLEDGE が提案されました。", width="800", height="169" %}
```

File: site/ja/docs/privacy-sandbox/proposal-lifecycle/index.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+通常、Chrome
+でのテストは、フラグを使うことでデベロッパーがローカルでテストできる機能から開始されます。そのため、デベロッパーはブラウザでフラグを有効にして試す必要があります。多くの場合、これらのコードは非常に新しいため、問題が発生する可能性があります。
+
+また、Google ではオリジン トライアルも実施しており、それぞれ実施期間と
+対象の Chrome ユーザーの人数が限定されています。オリジン
+トライアルは一般公開されており、すべてのデベロッパーが参加できます。参加方法は登録してサイトまたはサービスをオプトインするだけです。ここで
+Google
+は、何が成功し、何がうまくいかず、その差がどこにあるかについて、デベロッパーから実用的なフィードバックを得ています。
+
+{% Img src="image/C47gYyWYVMMhDmtYSLOWazuyePF2/s6gTNVydmZZmGyivBClC.avif", alt="ALT_TEXT_HERE", width="800", height="368" %}
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/proposal-lifecycle/index.md:95]:
```suggestion
{% Img src="image/VbsHyyQopiec0718rMq2kTE1hke2/vTqicAx2UauzkePJkM0A.png", alt="ALT_TEXT_HERE", width="800", height="368" %}
```

File: site/ja/docs/privacy-sandbox/proposal-lifecycle/index.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+[User-Agent Client Hints](https://web.dev/user-agent-client-hints/)（UA-CH）は
+2021 年に Chrome でリリースされました。これはプライバシー
+サンドボックスのワーク
+ストリームの一部であり、ブラウザのフィンガープリントなどの密かな追跡を制限します。
+
+Cookie と同様に、ユーザー
+エージェント（UA）文字列は初期のウェブ機能です。デフォルトでは、ユーザーのブラウザとデバイスに関する多くの情報が提供されるため、フィンガープリントに容易に利用できます。また、解析の際には悩みの種になりうる形式でもあります。
+
+{% Img src="image/C47gYyWYVMMhDmtYSLOWazuyePF2/mgc6Yud6cv4sosrObMKD.avif", alt="たとえば、「User-Agent: Mozilla/5.0 (Linux; Android 10; Pixel 3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4076.0 Mobile Safari/537.36」は非常に長く、フィンガープリントに使用される特定の詳細情報（デバイスの正確な機種、プラットフォームのバージョン、Chrome のバージョン全体など）を提供しています。", width="800", height="464" %}
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/proposal-lifecycle/index.md:132]:
```suggestion
{% Img src="image/VbsHyyQopiec0718rMq2kTE1hke2/AVzbV9HF0T0bm3buFjV6.jpg", alt="たとえば、「User-Agent: Mozilla/5.0 (Linux; Android 10; Pixel 3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4076.0 Mobile Safari/537.36」は非常に長く、フィンガープリントに使用される特定の詳細情報（デバイスの正確な機種、プラットフォームのバージョン、Chrome のバージョン全体など）を提供しています。", width="800", height="464" %}
```

File: site/ja/docs/privacy-sandbox/proposal-lifecycle/index.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+削減されたユーザー
+エージェントには、リクエストの送信元であるブラウザのブランドとメジャーバージョン（デスクトップまたはモバイル）、プラットフォームが含まれます。今後、ユーザーのデバイスや条件に関する特定の情報など、より多くのデータにアクセスする必要がある場合は、UA-CH
+を使用する必要があります。
+
+言い換えると、ユーザー エージェント
+データは「デフォルトで使用可能な」モデルから「リクエストに応じた」モデルに移行します。これはプライバシーに関する現在の優れた方法であり、Google
+が今後設定していきたいと考えているパターンです。
+
+{% Img src="image/C47gYyWYVMMhDmtYSLOWazuyePF2/UCXrDoPE25iNA5cIEJWX.avif", alt="ALT_TEXT_HERE", width="800", height="339" %}
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/proposal-lifecycle/index.md:142]:
```suggestion
{% Img src="image/VbsHyyQopiec0718rMq2kTE1hke2/ZsumGF9jzVb5yYL4QD3i.png", alt="ALT_TEXT_HERE", width="800", height="339" %}
```

**Review by agektmr — CHANGES_REQUESTED** [review]
Lots of avif resources from nowhere. Where are they come from?

File: site/ja/docs/privacy-sandbox/proposal-lifecycle/index.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+---
+
+[提案のライフサイクル](/docs/privacy-sandbox/proposal-lifecycle/)
+
+GitHub English source:
+[https://raw.githubusercontent.com/GoogleChrome/developer.chrome.com/main/site/en/docs/privacy-sandbox/proposal-lifecycle/index.md](https://raw.githubusercontent.com/GoogleChrome/developer.chrome.com/main/site/en/docs/privacy-sandbox/proposal-lifecycle/index.md)
+
+Google
+がどのようなプロセスで関係者と協力して、プライバシー保護技術についてディスカッション、テスト、導入するかについて。
+
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/proposal-lifecycle/index.md:16]: Line 9 to 15 are not part of the article. Please remove.

File: site/ja/docs/privacy-sandbox/proposal-lifecycle/index.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+_こちらのコンテンツの多くは、元々は [2021 年の Chrome Developer Summit のまとめ](/docs/privacy-sandbox/cds21-update/)の一部として共有されていたものです。_
+
+## [プライバシー サンドボックスの提案の目的](/docs/privacy-sandbox/proposal-lifecycle/#what-is-the-goal-of-the-privacy-sandbox-proposals)
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/proposal-lifecycle/index.md:19]: This link is not necessary. Please try to replicate the English version as much as possible. Other headers as well.

**Review by agektmr — CHANGES_REQUESTED** [review]

**Comment by mattletford** [comment]
I've applied the suggestions @agektmr

File: site/ja/docs/privacy-sandbox/proposal-lifecycle/index.md [hunk 7]
```diff
@@ -6,18 +6,8 @@ authors:
...
-
-GitHub English source:
-[https://raw.githubusercontent.com/GoogleChrome/developer.chrome.com/main/site/en/docs/privacy-sandbox/proposal-lifecycle/index.md](https://raw.githubusercontent.com/GoogleChrome/developer.chrome.com/main/site/en/docs/privacy-sandbox/proposal-lifecycle/index.md)
-
-Google
-がどのようなプロセスで関係者と協力して、プライバシー保護技術についてディスカッション、テスト、導入するかについて。
-
 _こちらのコンテンツの多くは、元々は [2021 年の Chrome Developer Summit のまとめ](/docs/privacy-sandbox/cds21-update/)の一部として共有されていたものです。_
 
-## [プライバシー サンドボックスの提案の目的](/docs/privacy-sandbox/proposal-lifecycle/#what-is-the-goal-of-the-privacy-sandbox-proposals)
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/proposal-lifecycle/index.md]: This shouldn't be removed.

File: site/ja/docs/privacy-sandbox/proposal-lifecycle/index.md (hunk 6 above)
Comment (agektmr) [reply site/ja/docs/privacy-sandbox/proposal-lifecycle/index.md]: This comment still stands. Please go over all headings, comparing with the English version and remove links where necessary.

Comment (agektmr) [reply site/ja/docs/privacy-sandbox/proposal-lifecycle/index.md]: What I meant is, to remove the link not to remove the heading itself.

**Comment by mattletford** [comment]
@agektmr thanks, I've removed the links from the headings

**Review by agektmr — APPROVED** [review]
LGTM

**Review by heyawhite — APPROVED** [review]

**Pull request merged**